third_party/airflow/examples/hello_armada.py

```diff
@@ -0,0 +1,95 @@
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from armada.operators.armada import ArmadaOperator
+
+from armada_client.k8s.io.api.core.v1 import generated_pb2 as core_v1
+from armada_client.k8s.io.apimachinery.pkg.api.resource import (
+    generated_pb2 as api_resource,
+)
+
+from armada_client.armada import (
+    submit_pb2,
+)
+
+from armada_client.client import ArmadaClient
+import grpc
+
+import pendulum
+from armada.operators.jobservice import JobServiceClient
+
+
+def submit_sleep_job():
+    """
+    This is a PodSpec definition that allows you to run sleep.
+    This returns an array of JobSubmitRequestItems that allows you
+    to submit to Armada.
+    """
+    pod = core_v1.PodSpec(
+        containers=[
+            core_v1.Container(
+                name="sleep",
+                image="busybox",
+                args=["sleep", "10s"],
+                securityContext=core_v1.SecurityContext(runAsUser=1000),
+                resources=core_v1.ResourceRequirements(
+                    requests={
+                        "cpu": api_resource.Quantity(string="120m"),
+                        "memory": api_resource.Quantity(string="510Mi"),
+                    },
+                    limits={
+                        "cpu": api_resource.Quantity(string="120m"),
+                        "memory": api_resource.Quantity(string="510Mi"),
+                    },
+                ),
+            )
+        ],
+    )
+
+    return [submit_pb2.JobSubmitRequestItem(priority=1, pod_spec=pod)]
+
+
+"""
```

Comment:
I am not sure how important this is to people, but because of [PEP 257](https://peps.python.org/pep-0257/), most IDE's are going to include these docstrings as part of the files module, rather than comments for the individual places where they are used.

This will have limited scope, but it could be annoying if an IDE decides to show all these docstrings when the user only expects to see the files/module docstring.

Reply:
@jayofdoom suggested docs in examples as a way to provide documentation so someone could understand how to write their own examples.  

Are you suggesting we use normal comments and not do strings.

Reply:
Yeah exactly. Just suggesting to use normal comments

Reply:
Eh, I don't think we should reduce the value of the comments because they show up in an ide.